xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -460,6 +460,7 @@ extern "C" {
     unsigned int    iMarginEnd;                                /*!< @brief (optional) if set, the backend ends the recording iMarginEnd minutes after endTime. */
     int             iGenreType;                                /*!< @brief (optional) genre type */
     int             iGenreSubType;                             /*!< @brief (optional) genre sub type */
+    const char *    strGenreDescription;                       /*!< @brief (optional) genre. Will be used only when iGenreType = EPG_GENRE_USE_STRING */
```

Comment:
This seems not to be used anywhere. So, either add an implementation to kodi pvr core or drop this line, please.